### ISSUE TITLE
About vitis ai and DPU running two xmodels

### ISSUE THREAD

**Issue by Jeromeysjd** [Xilinx/Vitis-AI#1429]
I'm using a kv260 FPGA to deploy neural networks, and I'm asking if it's possible to deploy two models of the same deep learning framework (.pth) on the kv260 if you use the vitis ai tool and a DPU computing unit, i.e., run two .xmodels on a single dpu?

**Comment by KrishnaGaihre** [comment]
Yes you can run two model or multi-ml model in KV260. Gstreamer approach allows to run multi ml model with multi input of data. You have to follow the VVAS (which provides gstreamer based flow for running ml models): https://xilinx.github.io/VVAS/main/build/html/docs/Embedded/smart_model_select.html


Regards,
Krishna@LogicTronix
